torch/_dynamo/variables/torch.py

```diff
@@ -172,6 +173,11 @@ def call_function(
     ) -> "VariableTracker":
         from . import ConstantVariable, GradModeVariable, TensorVariable
 
+        if inductor_config.triton.cudagraphs and self.value.__name__ in [
```

Comment:
This does not look like the right place to enter this code.

There are a couple of ways

1) At user level - use `torch._dynamo.disallow_in_graph` - https://github.com/pytorch/pytorch/blob/master/test/dynamo/test_misc.py#L1696-L1714. This would cause a graph break at dynamo-level w/o changing Dynamo code. This gives the control to user.

2) At inductor level - Check if the fx graph has an embedding_op, if it has skip cudagraphs. The relevant guarding is here - https://github.com/pytorch/pytorch/blob/master/torch/_inductor/compile_fx.py#L130-L133. This is not a good option if the graph is very large, and we have to skip cudagraphs for the whole graph just because of embedding_op.